.github/workflows/deploy.yaml

```
@@ -0,0 +1,42 @@
+name: Deploy to Firebase
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10
+
+      - name: Installing Dependencencies
+        run: npm install
+
+      - name: Building the project
+        run: npm run build
+
+      - name: Deploy to Firebase
+        uses: w9jds/firebase-action@master
```

I'd feel better not pulling directly from master.

```suggestion
        uses: w9jds/firebase-action@v1
```

---

Yup, I've actually pinned to `v1.3.0`